.gitignore

```diff
@@ -6,6 +6,7 @@
 *.log
 dist
 .idea
+mongo.js
```

Comment:
It is the js file that is generated after the tests insert.update and remove data from the collections. It can be used to populate a db  with data, if needed.